src/docs/ftw-styling/how-to-customize-ftw-styles/index.md

```diff
@@ -61,6 +83,35 @@ used inside some CSS rule.
 (Read more about CSS Properties from
 [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties))
 
+### customMediaQueries.css
+
+Breakpoints for media queries are defined in separate file.
+
+```css
+@custom-media --viewportSmall (min-width: 550px);
+@custom-media --viewportMedium (min-width: 768px);
+@custom-media --viewportLarge (min-width: 1024px);
+// etc.
+```
+
+These custom media query breakpoints can be used in a similar way as CSS
+Properties. However, these variable are converted to real media queries
```

Comment:
Double-checking (I'm going to add a link here) that CSS Properties refer to this:
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties
@Gnito